frontend/src/app/modules/registration/registration-form/registration-form.component.ts

```diff
@@ -124,12 +160,41 @@ export class RegistrationFormComponent extends BaseComponent implements OnInit {
         return null;
     };
 
+    next() {
+        if (this.step === this.stepRegistration.First) {
+            this.personal_step = true;
+            if (this.personalDetail.invalid) { return; }
+            this.step += 1;
+        }
+    }
+
+    previous() {
+        this.step -= 1;
+        if (this.step === this.stepRegistration.First) {
+            this.personal_step = false;
```

Comment:
You can rewrite that valiable to getter, where you will check `this.step === this.stepRegistration.First`

Reply:
No, this variable should be here

Reply:
What's the point in this variable?

Reply:
She is responsible for the tabs. And for the content that will be show.

Reply:
And then what is this.step responsible for? Why do we need 2 variables that are responsible for tabs?

Reply:
I need to do checks in ngIf. Without this.step, it won't work
![image](https://user-images.githubusercontent.com/78735694/131618856-e2f6cca3-9099-4c65-a495-85afc5ea5bee.png)
![image](https://user-images.githubusercontent.com/78735694/131618916-88ada6c7-5af4-4743-9531-8f9890ee1fce.png)

Reply:
@tatianahutii  Ok?
![image](https://user-images.githubusercontent.com/78735694/131627923-38c889e3-cfed-43ff-afa2-64ccb7245499.png)

Reply:
Ok, but I don't understand why you use personalStep and this.step simultaneously. Please rewrite to use only this.step.